### PR TITLE
remove Python <3.6 from setup.py

### PR DESCRIPTION
**Pull request by russoz**
Hi, I am a maintainer of Ansible's community.general collection (https://github.com/ansible-collections/community.general/) and I have been giving some love to the proxmox modules in the last couple of days. In doing so, I started looking into the code of proxmoxer, and I noticed that, despite using f-strings, `setup.py` still claims it is compatible with Python 2.7.

Hence, I am submitting this PR to set the minimum version at Python 3.6 when f-strings were introduced. I have not examined the code further to determine if other dependencies to higher versions of Python are in use. It might be good to check on that though.

Thanks
Alex

### CODE REVIEW

**Comment by russoz** [comment]
BTW, I did not do my homework looking for contributors' guidelines, so if this PR needs to comply with some rules, would you please advise where they are and I will be happy to comply.

File: setup.py [hunk 1]
```diff
@@ -41,10 +41,6 @@
         "Intended Audience :: System Administrators",
         "License :: OSI Approved :: MIT License",
         "Operating System :: OS Independent",
-        "Programming Language :: Python :: 2.7",
-        "Programming Language :: Python :: 2",
-        "Programming Language :: Python :: 3.4",
-        "Programming Language :: Python :: 3.5",
         "Programming Language :: Python :: 3.6",
```
Comment (jhollowe) [inline setup.py:44]:
```suggestion
```

also remove 3.6 as we are only supporting python versions still in support. 

(3.7 just went out of support, but there isn't anything that really needs to move off of it so we can continue supporting it until a new feature requires only newer versions.)

**Comment by jhollowe** [comment]
Thanks for catching this! If you have any questions, feel free to make an issue or reach out to me directly!

**Comment by russoz** [comment]
Thank you @jhollowe ! I am not an user of proxmox myself, but will reach out if needed :-)

**Pull request merged**